go/obscuronode/enclave/core/test_utils.go

```diff
@@ -7,9 +7,7 @@ import (
 
 	"github.com/obscuronet/obscuro-playground/go/obscuronode/nodecommon"
 
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/rlp"
 )
 
 // Creates a dummy L2Tx for testing
```

Comment:
Should this go to the datagenerator package ?

Reply:
donno, this is your class :)

Reply:
I think having all random data generators in the same package is pretty helpful